Kconfig.zephyr

```diff
@@ -167,6 +167,22 @@ config LINKER_SORT_BY_ALIGNMENT
 	  in decreasing size of symbols. This helps to minimize
 	  padding between symbols.
 
+config HAS_SRAM_OFFSET
+	bool
+	help
+	  This option is selected by targets that require SRAM_OFFSET.
```

Comment:
Do you need this hidden option ? I don't see anywhere in this pr an usage of this

Reply:
This is being used in the commit dropping usage of `CONFIG_X86_KERNEL_OFFSET` in favor of `CONFIG_SRAM_OFFSET`.